Base/Logger/Logger.Test/FileLoggerTests.cpp

```diff
@@ -135,7 +135,7 @@ TEST_CASE("Logging and throw exception method")
     std::exception* someException;
 
 #ifdef _WIN32
-    someException = new std::exception("Something wrong in std::exception");
+    someException = new std::exception("Test std::exception");
 #else // bacause std::exception has no _Data for std::exception.what() in GCC and CLANG
```

Comment:
typo 'bacause'

Reply:
This missprint has been corrected with the last [commit](https://github.com/f1nal3/Juniorgram/pull/613/commits/533c61ff64ddf42c1f9fcd2a41a1faebce09588f)